### PR TITLE
Access-Control-Allow-Max-Age is actually Access-Control-Max-Age

### LINKED ISSUE #9951: CORS config "maxAge" sends wrong response header
**Having the following cors config in server.xml**

` 
<cors domain="/" 
          allowedOrigins="*"
          allowedMethods="GET,DELETE,POST,PUT,OPTIONS"
          allowedHeaders="origin,accept,content-type,authorization,if-match"
          exposeHeaders="etag"
          allowCredentials="true"
          maxAge="600" />
`

**Liberty sends the following response headers:**

`
Access-Control-Allow-Credentials: true
Access-Control-Allow-Origin: http://localhost:3000
Access-Control-Allow-Max-Age: 600
Access-Control-Allow-Methods: GET, DELETE, POST, PUT, OPTIONS
Access-Control-Allow-Headers: origin, accept, content-type, authorization, if-match, accept-language`

**The browser does not cache the response, because**

`Access-Control-Allow-Max-Age: 600 `

**should be** 

`Access-Control-Max-Age: 600`

([https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age))

OpenLiberty Version: (Open Liberty 19.0.0.11/wlp-1.0.34.cl191120191031-0300) on Eclipse OpenJ9 VM, version 11.0.2+9

### PR DESCRIPTION
**Pull request by pugmas**
Resolves #9951

### CODE REVIEW

**Comment by navidsh** [comment]
#build

**Comment by navidsh** [comment]
Hi @pugmas, before we can merge this PR, we need you to sign a Contributor License Agreement (CLA). You can read about how to do it in [Contributor License Agreement](https://github.com/OpenLiberty/open-liberty/blob/integration/CONTRIBUTING.md#contributor-license-agreement).

**Comment by pugmas** [comment]
Is on the way.

**Comment by pugmas** [comment]
#build

**Comment by navidsh** [comment]
Thanks @pugmas. I will request a #build on your behalf.

File: dev/com.ibm.ws.webcontainer.cors/src/com/ibm/ws/webcontainer/cors/CorsHelper.java [hunk 1]
```diff
@@ -67,7 +67,7 @@
     private static final String RESPONSE_HEADER_ACCESS_CONTROL_EXPOSE_HEADERS = "Access-Control-Expose-Headers";
 
     /** Optional. Response to preFlight(OPTIONS). Indicates how long the results of a preFlight request can be cached. */
-    private static final String RESPONSE_HEADER_ACCESS_CONTROL_MAX_AGE = "Access-Control-Allow-Max-Age";
```
Comment (navidsh) [inline dev/com.ibm.ws.webcontainer.cors/src/com/ibm/ws/webcontainer/cors/CorsHelper.java]: Please update the [copyright header](https://github.com/OpenLiberty/open-liberty/pull/9952/files#diff-1c806706d819c2cd9f8ff364d77a9133R2) in `Copyright (c) 2015, 2017...` to `Copyright (c) 2015, 2019...`

File: dev/com.ibm.ws.webcontainer.cors/test/com/ibm/ws/webcontainer/cors/CorsHelperTest.java [hunk 2]
```diff
@@ -56,7 +56,7 @@
     private static final String RESPONSE_HEADER_ACCESS_CONTROL_EXPOSE_HEADERS = "Access-Control-Expose-Headers";
 
     /** Optional. Response to preFlight(OPTIONS). Indicates how long the results of a preFlight request can be cached. */
-    private static final String RESPONSE_HEADER_ACCESS_CONTROL_MAX_AGE = "Access-Control-Allow-Max-Age";
+    private static final String RESPONSE_HEADER_ACCESS_CONTROL_MAX_AGE = "Access-Control-Max-Age";
```
Comment (navidsh) [inline dev/com.ibm.ws.webcontainer.cors/test/com/ibm/ws/webcontainer/cors/CorsHelperTest.java]: Please update [copyright header](https://github.com/OpenLiberty/open-liberty/pull/9952/files#diff-6a558ddb6ca3e5a821fcd0281ee1d5fbR2) to `Copyright (c) 2011, 2019 IBM...`

**Comment by navidsh** [comment]
#build

**Comment by navidsh** [comment]
#run-libby-bot

**Comment by navidsh** [comment]
I just checked the final build results. There is 1 intermittent failure not related to the changes made in this PR.

**Pull request merged**